src/test/java/com/github/herrschwarz/status4spring/cache/GuavaCacheStatsProviderTest.java

```diff
@@ -0,0 +1,106 @@
+package com.github.herrschwarz.status4spring.cache;
+
+import com.google.common.cache.Cache;
+import com.google.common.collect.ImmutableList;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.cache.guava.GuavaCache;
+import org.springframework.cache.guava.GuavaCacheManager;
+
+import java.util.Collection;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GuavaCacheStatsProviderTest {
+    private static final String CACHE_NAME = "test";
+    private static final String CACHE_TYPE = "Guava Cache";
+    private static final long NUMER_OF_ENTRIES = 2L;
+    private Collection<String> cacheNames = ImmutableList.of(CACHE_NAME);
+
+    @Mock
+    private GuavaCacheManager cacheManager;
+    @Mock
+    private Cache<Object, Object> cache;
+
+    private GuavaCache guavaCache;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'guavaCache' could be replaced by a local variable.](https://www.codacy.com/app/felix_3/status4spring/file/2642427046/issues/source?bid=3309767&fileBranchId=3321843#l33)